test/dialect/AIEVec/Transform/invalid.mlir

```diff
@@ -0,0 +1,182 @@
+// RUN: aie-opt %s -test-transform-dialect-interpreter -split-input-file -verify-diagnostics
```

Comment:
Do we need file check in this test case?

Reply:
No, `-verify-diagnostics` is the option that consumes the `expected-error` directives.